python/dgl/distributed/rpc_client.py

```diff
@@ -163,7 +163,7 @@ def connect_to_server(ip_config, num_servers, max_queue_size=MAX_QUEUE_SIZE, net
         server_ip = addr[1]
         server_port = addr[2]
         while not rpc.connect_receiver(server_ip, server_port, server_id):
-            time.sleep(1)
+            time.sleep(3)
```

Comment:
does connecting the receiver generate the error messages if the server isn't available? if so, sleeping 3 seconds will still print many error messages.

Reply:
`DLOG` instead of `LOG` is used in `tp_communicator`, so for normal run, `Failed to connect...` will not be printed out unless `DMLC_LOG_DEBUG=1` is defined.